include/uapi/sound/sof/tokens.h

```diff
@@ -204,4 +204,18 @@
 #define SOF_TKN_AMD_ACPI2S_CH			1701
 #define SOF_TKN_AMD_ACPI2S_TDM_MODE		1702
 
+/* MODULE PIN AUDIO FORMAT */
+#define SOF_TKN_COMP_PIN_FORMAT_RATE			2000
+#define SOF_TKN_COMP_PIN_FORMAT_BIT_DEPTH		2001
+#define SOF_TKN_COMP_PIN_FORMAT_VALID_BIT		2002
+#define SOF_TKN_COMP_PIN_FORMAT_CHANNELS		2003
+#define SOF_TKN_COMP_PIN_FORMAT_CH_MAP			2004
+#define SOF_TKN_COMP_PIN_FORMAT_CH_CFG			2005
+#define SOF_TKN_COMP_PIN_FORMAT_INTERLEAVING_STYLE	2006
+#define SOF_TKN_COMP_PIN_FORMAT_FMT_CFG			2007
```

Comment:
what is a fmt_cfg, givent that we've already listed all possible options for the format?